Support optional comment for service rule.

AAbouZaid opened the pull request:
I added support for optional comment in case a comment is needed to be added.

I did test it on:
```
Ubuntu 14.04
SaltStack: Carbon - 2016.11.0
```

twellspring commented:
Looks like this is also pulling in the changes from [PR 8](https://github.com/saltstack-formulas/iptables-formula/pull/8).  Probably should test/merge that before this one.

AAbouZaid commented:
@twellspring I've just switched the branch so this PR is not valid anymore.

Actually I moved to @stenstad fork which is based on yours, because this official repo looks like not maintained anymore! (there are PRs since 2015!)

I have more fixes and I will push them asap to stenstad fork :-)

AAbouZaid commented:
I really believe in upstream, but if upstream is dead, someone should take over the old upstream or find another live one :-)

myoung34 commented:
@AAbouZaid is this tested? on both ubuntu and redhat?

AAbouZaid commented:
@myoung34 Sorry I did switch the base branch, this another branch. I'm fixing this now.

AAbouZaid commented:
Here we are ... it's ready now, and it tested with Ubuntu 14.04 and CentOS 7.

There is another issue related to `network` section which is not exists anymore in code but in still in `pillar.example` file.
That one:
```
  whitelist:
    networks:
      ips_allow:
        - 10.0.0.0/8
```
But that should have another PR :)

myoung34 commented:
@AAbouZaid wanna make a PR for that?

myoung34 commented:
Merging because there has to be a bit of trust involved, PR's shouldn't sit for that long. If this PR introduces an issue we can handle that in the next one.

myoung34 commented:
@AAbouZaid can you also do me a favor and let me know if any of the issues/PR's that are still open here can be closed as a result of this one?

AAbouZaid commented:
@myoung34 Sure, thanks :-)
